Source/Private/ASDisplayNode+AsyncDisplay.mm

```diff
@@ -347,6 +349,12 @@ - (void)displayAsyncLayer:(_ASDisplayLayer *)asyncLayer asynchronously:(BOOL)asy
         layer.contents = (id)image.CGImage;
       }
       [self didDisplayAsyncLayer:self.asyncLayer];
+      
+      if (_flags.rasterizesSubtree) {
```

Comment:
Since we no longer hold the instance lock by the time this `completionBlock` is executed, I think we should either grab this flag at the beginning of this method and use it here, or call `self.rasterizesSubtree` here.